llvm/lib/CodeGen/TargetLoweringBase.cpp

```diff
@@ -956,6 +956,10 @@ void TargetLoweringBase::initActions() {
   // This one by default will call __clear_cache unless the target
   // wants something different.
   setOperationAction(ISD::CLEAR_CACHE, MVT::Other, LibCall);
+
+  // By default, STACKADDRESS nodes are expanded to STACKSAVE nodes.
```

Comment:
```suggestion
  // By default, STACKADDRESS nodes are expanded like STACKSAVE nodes.
```
(We don't actually expand STACKADDRESS -> STACKSAVE, we just use the same legalization approach.)